src/dstack/_internal/core/backends/aws/compute.py

```diff
@@ -706,11 +755,13 @@ def _supported_instances(offer: InstanceOffer) -> bool:
         "g4dn.",
         "g5.",
         "g6.",
+        "g6e.",
         "gr6.",
         "p3.",
         "p4d.",
         "p4de.",
         "p5.",
+        "p5e",
```

Comment:
"p5e" -> "p5e." for consistency?

Also note that p5e instances won't work with dstack now because gpuhunt doesn't include these offers – AWS stopped to return pricing for p5e in their pricing csv for some reason. Apparently, it's because p5e is only available via Capacity Blocks.

Reply:
Right, I'll remove it from the filter then.

Reply:
@solovyevt, let us know if supporting "p5e" instances is critical to you. You can file an issue, and we'll prioritize it.